src/AzureRenderManager/WebApp.Tests/Authorization/AuthorizationManagerTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WebApp.Authorization;
+using WebApp.Config;
+using WebApp.Models.Environments;
+using WebApp.Tests.Fakes;
+using Xunit;
+
+namespace WebApp.Tests.Authorization
+{
+    public class AuthorizationManagerTests
+    {
+        [Fact]
+        public async Task SubscriptionOwnerIsEnvOwner()
+        {
+            var arm = new FakeAzureResourceProvider();
+            var graphProvider = new AuthorizationManager(null, null, arm, null);
+            var env = RenderingEnvironment;
+
+            arm.UserPermissions.Add(new UserPermission {
+                ObjectId = Guid.NewGuid().ToString(),
+                Scope = $"/subscriptions/{env.SubscriptionId}",
+                Role = "Owner"
+            });
+
+            var perms = await graphProvider.ListUserPermissions(env);
+
+            Assert.Single(perms);
```

Comment:
Suggestion: something useful is that `Assert.Single` returns the item, so you can write instead:

```
var perm = Assert.Single(perms);
Assert.Equal(PortalRole.Owner.ToString(), perm.Role);
```